cnf-certification-test/networking/netutil/netutil.go

```diff
@@ -77,3 +77,17 @@ func GetListeningPorts(cut *provider.Container) (map[PortInfo]bool, error) {
 
 	return parseListeningPorts(outStr)
 }
+
+func GetSSHDaemonPort(cut *provider.Container) (string, error) {
+	const findSSHDaemonPort = "grep -w 'Port' /etc/ssh/sshd_config"
```

Comment:
```suggestion
	const findSSHDaemonPort = "sudo netstat -tpln | grep sshd | head -1 | awk '{ print $4 }' | awk -F : '{ print $2 }'"
```

Reply:
I would have loved to use netstat, but this is deprecated in favour of the suite iproute2.

Reply:
```suggestion
	const findSSHDaemonPort = "sudo ss -tpln | grep sshd | head -1 | awk '{ print $4 }' | awk -F : '{ print $2 }'"
```